### PR TITLE
Readme update and examples folder

### PR DESCRIPTION
**Pull request by isaacaflores2**
Hey Craig, 

I just wanted to update the readme to specify that the public property names must match the names of the column headers in the worksheet. I did not notice this requirement at first, so I just wanted to make it a little clearer. 

Additionally, I added an examples folder that contains the Person class and example code to read and write. 

Thanks btw, this project made it really easy to read a worksheet!

### CODE REVIEW

File: examples/Person.cs [hunk 1]
```diff
@@ -0,0 +1,17 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Excel.IO.examples
```
Comment (craigomatic) [inline examples/Person.cs]: Could we capitalise Examples:

```suggestion
namespace Excel.IO.Examples
```

File: examples/ReadExample.cs [hunk 2]
```diff
@@ -0,0 +1,21 @@
+﻿using Excel.IO;
+using System;
+using System.Collections.Generic;
+
+namespace Excel.IO.examples
```
Comment (craigomatic) [inline examples/ReadExample.cs]: Ditto:
```suggestion
namespace Excel.IO.Examples
```

**Review by craigomatic — COMMENTED** [review]
Thanks for taking the time to submit this PR!

Some suggestions:

1. Consider removing personal info from the xlsx file (you can do this via Check For Issues -> Inspect Document in Excel)
2. Could we include a csproj for your examples? Console app with a Program.cs that runs the Read/Write examples? 
3. Could we also do relative path for the xlsx file references?

File: examples/ReadExample.cs (hunk 2 above)
Comment (isaacaflores2) [reply examples/ReadExample.cs]: Craig, 
1) I removed the personal info
2) I have included the full solution, so someone can easily run the program. 
3) I added the relative path. 
4) I updated the namespace

Let me know if you would like me to make any other updates.

**Comment by craigomatic** [comment]
Looks good! Last small request would be to switch the example project to target net core 3.0 while 3.1 is in preview:

`<TargetFramework>netcoreapp3.0</TargetFramework>`

File: examples/ReadExample.cs (hunk 2 above)
Comment (isaacaflores2) [reply examples/ReadExample.cs]: Craig, 

Cool! I just switched the target framework.

**Pull request merged**